source/core/NumericString.cpp

```diff
@@ -2090,7 +2090,7 @@ struct ByteArrayToStringParamBlock : InstructionCore
 static bool CheckUnsupportedEncodingError(ErrorCluster* errorCluster, UInt16 stringEncoding, TypedArrayCoreRef outputArray) {
     if (stringEncoding != 0) {
         if (errorCluster != nullptr) {
-            errorCluster->SetErrorAndAppendCallChain(true, 1, "Encoding type not supported");
+            errorCluster->SetErrorAndAppendCallChain(true, 1396, "Encoding type not supported");
```

Comment:
Please create an enum for the error code above the function definition. Something like this:

// Encoding error codes
enum { kCannotConvertErr = 1396 };

And use kCannotConvertErr instead of the "1396" hardcode constant.

We have been updating these hardcode numbers to something more meaningful as we change the code.